processor/src/main/java/com/squareup/sqlbrite/SqlBriteProcessor.java

```diff
@@ -0,0 +1,152 @@
+package com.squareup.sqlbrite;
+
+import com.squareup.javapoet.ClassName;
+import com.squareup.javapoet.FieldSpec;
+import com.squareup.javapoet.JavaFile;
+import com.squareup.javapoet.MethodSpec;
+import com.squareup.javapoet.TypeName;
+import com.squareup.javapoet.TypeSpec;
+import com.squareup.sqlbrite.annotation.Column;
+import com.squareup.sqlbrite.annotation.Table;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.Filer;
+import javax.annotation.processing.Messager;
+import javax.annotation.processing.ProcessingEnvironment;
+import javax.annotation.processing.RoundEnvironment;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.Modifier;
+import javax.lang.model.element.PackageElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.element.VariableElement;
+import javax.lang.model.util.ElementFilter;
+import javax.tools.Diagnostic;
+
+public class SqlBriteProcessor extends AbstractProcessor {
+  private static ClassName CONTENT_VALUES_CLASS = ClassName.get("android.content", "ContentValues");
+
+  private Filer filer;
+  private Messager messager;
+
+  @Override
+  public synchronized void init(ProcessingEnvironment processingEnv) {
+    super.init(processingEnv);
+    filer = processingEnv.getFiler();
+    messager = processingEnv.getMessager();
+  }
+
+  @Override
+  public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+    for (Element annotated : roundEnv.getElementsAnnotatedWith(Table.Builder.class)) {
+      processType(annotated);
+    }
+    return false;
+  }
+
+  private void processType(Element element) {
+    if (element.getKind() != ElementKind.INTERFACE) {
+      messager.printMessage(Diagnostic.Kind.ERROR, "Can only annotate " + Table.Builder.class +
```

Comment:
`"Can only annotate @" + Table.Builder.class.getSimpleName()`, otherwise you get "class blah.blah.blah.Blah" as the output.